packages/volto-hydra/src/components/Iframe/View.jsx

```diff
@@ -5,26 +5,22 @@ import Cookies from 'js-cookie';
 import './styles.css';
 
 const Iframe = () => {
-  const [url, setUrl] = useState('');
-
-  const [src, setSrc] = useState('');
   const history = useHistory();
   const token = useSelector((state) => state.userSession.token);
-
+  const form = useSelector((state) => state.form.global);
   const getDefualtUrlFromEnv = () =>
     process.env['RAZZLE_DEFAULT_IFRAME_URL'] ||
     (typeof window !== 'undefined' && window.env['RAZZLE_DEFAULT_IFRAME_URL']);
```

Comment:
I didn't catch this before. This function shouldn't be inside the React component. It doesn't access any props or state

Reply:
fixed in [other PR](https://github.com/collective/volto-hydra/pull/60/files#diff-58df3f36ad1434c88a8d92d9d9724a5c5c6bddb12f68fa2ef73bb64c0ad99a43)